index.js

```diff
@@ -759,3 +759,28 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+	var negatedMethods = {};
+	Object.keys(source).forEach(function (key) {
```

Comment:
I'll switch to using for-each.

I don't think the brute force approach is better, since it introduces more potential points of failure as the library is expanded. My implementation is a catch-all approach that ensures all methods are negated with only a single point of failure.